ui/app/src/components/DeletePluginDialog/UninstallPluginDialogBody.tsx

```diff
@@ -0,0 +1,148 @@
+import { Box, ListItem, ListItemText, styled, TextField, Theme } from '@mui/material';
+import Typography from '@mui/material/Typography';
+import List from '@mui/material/List';
+import ItemDisplay from '../ItemDisplay';
+import { SxProps } from '@mui/system';
+
+const Strong = styled('strong')``;
+
+function getStyles(sx: UninstallPluginDialogBodyPartialSx): UninstallPluginDialogBodyFullSx {
+  return {
+    content: {
+      background: (theme) => theme.palette.background.paper,
+      ...sx?.content
+    },
+    semiBold: {
+      fontWeight: 600,
+      ...sx?.semiBold
+    },
+    loadingStateWrapper: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      right: 0,
+      bottom: 0,
+      display: 'flex',
+      background: 'rgba(255,255,255,0.7)',
+      ...sx?.loadingStateWrapper
+    }
+  } as Record<UninstallPluginDialogBodyClassKey, SxProps<Theme>>;
+}
+
+export function UninstallPluginDialogBody(props: UninstallPluginDialogBodyProps) {
+  const { onCloseButtonClick, resource, pluginId, onSubmit: onSubmitProp, password = 'uninstall', submitting } = props;
+  const data = resource.read();
+  const hasUsages = data.length > 0;
+  const [confirmPasswordPassed, setConfirmPasswordPassed] = useState(false);
+  const [passwordFieldValue, setPasswordFieldValue] = useState('');
+  const sx = getStyles(props.sx);
+  useEffect(() => {
+    setConfirmPasswordPassed(passwordFieldValue.toLowerCase() === password.toLowerCase());
+  }, [password, passwordFieldValue]);
+  const onSubmit = (e) => {
+    e.preventDefault();
+    if (confirmPasswordPassed || !hasUsages) {
+      onSubmitProp?.();
+    }
+  };
+
+  return (
+    <>
+      <DialogBody>
+        {hasUsages ? (
+          <>
+            <Alert variant="outlined" severity="warning" icon={false}>
+              <FormattedMessage
+                id="uninstallPluginDialog.reviewDependenciesMessage"
+                defaultMessage={'Please review the dependant items of "{pluginId}"'}
+                values={{
+                  pluginId
+                }}
+              />
+            </Alert>
+            <List>
+              {data.map((item) => (
+                <ListItem key={item.id} divider sx={sx.content}>
+                  <ListItemText
+                    primary={<ItemDisplay item={item} showNavigableAsLinks={false} />}
+                    secondary={<Typography variant="body2" color="textSecondary" children={item.path} />}
+                  />
+                </ListItem>
+              ))}
+            </List>
+            <Alert severity="warning">
+              <FormattedMessage
+                id="uninstallPluginDialog.typePassword"
+                defaultMessage={'Type the word "<b>{password}</b>" to confirm the deletion of the plugin.'}
+                values={{
+                  password,
+                  b: (message) => {
+                    return <Strong sx={sx.semiBold}>{message}</Strong>;
```

Comment:
Strong is not a component, why is it capitalised? Are you sure this works?
```suggestion
                    return <strong sx={sx.emphasisedText}>{message}</strong>;
```

Reply:
Changed it to use `<Box component="strong" ...>`